pkg/util/cache/lruexpirecache.go

```diff
@@ -23,6 +23,7 @@ import (
 	"github.com/golang/groupcache/lru"
 )
 
+// 1. ttlcache 2. 同时是个lru cache 3.ttl时间不能修改
```

Comment:
Comments in english, please.

(sorry, but it is the most commonly used language amongst contributors)